vendor/github.com/portworx/sched-ops/k8s/storage/storageclass.go

```diff
@@ -34,6 +36,14 @@ const (
 	defaultStorageclassAnnotationKey = "storageclass.kubernetes.io/is-default-class"
 )
 
+// GetAllStorageClasses returns all storageClasses.
```

Comment:
@mkoppal-px take a look at this and let us know if you feel we need to raise it in Sched ops as well.

Reply:
@ak-px We should not be making changes here. If you need it here then it has to go in sched-ops